packages/chat-core-zendesk/src/infra/ChatCoreZendeskImpl.ts

```diff
@@ -63,7 +63,12 @@ export class ChatCoreZendeskImpl {
    * mode on the first invocation. Subsequent calls to this method will create a
    * new conversation session.
    */
-  async init(messageRsp: MessageResponse): Promise<void> {
+  async init(messageRsp: MessageResponse): Promise<string> {
+    await this.initializeZendeskSdk();
+    return await this.createZendeskConversation(messageRsp);
```

Comment:
nit: you can just return directly instead of doing `return await`, which created an intermediate promise we don't need